miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -46,6 +46,17 @@
         <li><a href="<c:url value="/miso/studies"/>">Studies</a></li>
         <li><a href="<c:url value="/miso/printers"/>">Printers</a></li>
     </ul>
+
+    <c:if test="${detailedSample}">
+      <h2>Institute Defaults</h2>
```

Comment:
This doesn't show up on most pages - only on the main menu, and the pages that the list contains from what I noticed. If you go to the Samples page, this part of the menu disappears, for instance

Reply:
I'm not sure what do do about this. This value isn't consistently provided to every page. I haven't been able to find a way to set it globally. Thoughts?

Reply:
You mean `detailedSample`? Shouldn't that come from `constants.js` now and be available everywhere?

Reply:
That's in the JavaScript. This happens in the JSP.

Reply:
Duh me... Well another option is to use a Controller Advice. Add a class with `@ControllerAdvice` and a `public boolean isDetailedSample()` method in it with `@ModelAttribute` and it will be added to every page